src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button({ text }) {
+  return (
+    <button type="button">{text}</button>
+  );
+}
+
+Button.propTypes = {
+  text: PropTypes.string.isRequired,
+};
+
+export default Button;
```

Comment:
- [ ] Good job implementing the `Button` child component, however, we have an error that has been introduced to the console showing us that we have an invalid prop `text` of type `number` supplied to `Button`, when what is expected is a `string`. Kindly fix this error.

Before child component implementation
![image](https://github.com/kazaneza/math-magician/assets/81932478/f10bc14f-db2d-40df-9b5d-443ab725d09f)

After
![image](https://github.com/kazaneza/math-magician/assets/81932478/47369af4-898f-4d2d-910f-1dd3b6c65b52)